foodsaving/conversations/api.py

```diff
@@ -82,20 +74,16 @@ def email_notifications(self, request, pk=None):
         return Response(serializer.data)
 
 
-class ConversationMessageViewSet(
-    mixins.CreateModelMixin,
-    mixins.ListModelMixin,
-    GenericViewSet
-):
+class ConversationMessageViewSet(mixins.CreateModelMixin, mixins.ListModelMixin, GenericViewSet):
```

Comment:
I think we should go for the multiline version, putting a comma after the last entry.